Assets/Scripts/Environment/FrustumCulling.cs

```diff
@@ -36,7 +36,23 @@ private void UpdateCullAbles()
             
             for (int i = 0; i < length; i++)
             {
-                _cullAbleObjects[i].SetActive(GeometryUtility.TestPlanesAABB(planes, _cachedRenderers[i].bounds));
+                bool isVisible = GeometryUtility.TestPlanesAABB(planes, _cachedRenderers[i].bounds);
+                if (isVisible)
```

Comment:
```suggestion
                bool isVisible = GeometryUtility.TestPlanesAABB(planes, _cachedRenderers[i].bounds);
                
                if (isVisible)
```
Styling van code is belangrijk.